### ISSUE TITLE
including import paths for packages 

### ISSUE THREAD

**Issue by kraxli** [landaire/deoplete-d#3]
Hi, thanks for deoplete-d which works well for std functions! 

However, I am wondering how to include the "include-paths" for other dub packages. Auto-completion works for std (phobos) functions only. In vim-dutyl I can for example specify other include paths aside the phobos one.

Thanks
Dave

**Comment by landaire** [comment]
deoplete-d currently does not parse any info from `dub`. An easy solution would be to run `dub describe` in the project directory and add those import paths. Let me see what I can do.

**Comment by landaire** [comment]
Could you try out the `dub-based-imports` branch I just pushed up and let me know how it works for you? Thanks.

**Comment by kraxli** [comment]
thanks for coming back on this so quickly! I tried the dub-based-imports branch but do not recognize any changes in the auto-completion behavior. Do I need to open the app.d file in a special way? How could I deal with just a simple d-source-code-file not related to a dub-project?

**Comment by landaire** [comment]
I was under the assumption that you're working in a dub-based project. I could also add support for non-dub projects by listing out dub packages (`dub list` I believe is the command) but that introduces issues with having multiple versions installed... let me play around with it tonight and I'll post back with an update.

**Comment by kraxli** [comment]
right, sometimes I use dub-projects but often I just have simple d-files with some import statements. Is there an option to manually feed an import path to deoplete-d as for example for dutyls? This is a bit painful but a first hack.
